tests/flobrij.ts

```diff
@@ -9,99 +9,97 @@ describe('flobrij', () => {
   const program = anchor.workspace.Flobrij as Program<Flobrij>;
 
   const EMAIL = 'test@test.com';
-  const TEST_AMOUNT = 5000;
+  const TEST_AMOUNT = new anchor.BN(5000);
   const EXP_HOURS = 8760;
-  const TEST_CREATOR = anchor.web3.Keypair.generate();
+  const TEST_RECIPIENT = anchor.web3.Keypair.generate();
   const AIRDROP_AMOUNT = 1000000000;
 
-  it('Create a receipt', async () => {
-    const receipt = anchor.web3.Keypair.generate();
-    const fakeTransaction = anchor.web3.Keypair.generate();
-    /*
-      transaction: Pubkey, 
-      recipient: Pubkey, 
-      email: String, 
-      amount: u32, 
-      expiration_hours: u16
-    */
-    const tx = await program.rpc.createReceipt(
-      fakeTransaction.publicKey,
-      TEST_CREATOR.publicKey,
-      EMAIL,
-      TEST_AMOUNT,
-      EXP_HOURS,
-      {
-        accounts: {
-          // Accounts here...
-          receipt: receipt.publicKey,
-          payer: program.provider.wallet.publicKey,
-          systemProgram: anchor.web3.SystemProgram.programId,
-        },
-        signers: [
-          // Key pairs of signers here...
-          receipt,
-        ],
-      }
+  it('should pay creator and create a receipt for patron', async () => {
+    // create new wallet for Creator
+    const newProvider = new anchor.Provider(
+      anchor.getProvider().connection,
+      new anchor.Wallet(TEST_RECIPIENT),
+      {}
+    );
+
+    // add some SOL
+    const signature = await newProvider.connection.requestAirdrop(
+      newProvider.wallet.publicKey,
+      AIRDROP_AMOUNT
+    );
+    await newProvider.connection.confirmTransaction(signature);
+
+    const creatorBalanceInitially = await newProvider.connection.getBalance(
+      newProvider.wallet.publicKey
     );
 
+    const receipt = anchor.web3.Keypair.generate();
+
+    const tx = await program.rpc.createReceipt(EMAIL, TEST_AMOUNT, EXP_HOURS, {
+      accounts: {
+        receipt: receipt.publicKey,
+        payer: program.provider.wallet.publicKey,
+        recipient: newProvider.wallet.publicKey,
+        systemProgram: anchor.web3.SystemProgram.programId,
+      },
+      signers: [receipt],
```

Comment:
Nit: Shouldn't the payer be a signer here?

Reply:
Anchor always add the payer signer automatically from what I read, so don't need to add explicitly.